src/test/resources/bundles/userCardExamples3/config.json

```diff
@@ -14,7 +14,8 @@
       "color": "#0070C0",
       "userCard" : {
         "template" : "usercard_task",
-        "lttdVisible" : false
+        "lttdVisible" : false,
+        "recipientVisible" : false
```

Comment:
I understand why we need a test with an empty recipientList, so OK for modifying the corresponding template under API, but not on this one because otherwise it makes the task usercard a bit useless to play with as it's only received by the sender themselves.